package.json

```diff
@@ -5,16 +5,17 @@
   "description": "Call Of Duty",
   "main": "server.js",
   "scripts": {
-    "test": "logLevel=silent vitest",
+    "test": "vitest",
```

Comment:
and the logs? you can use env.test if you like

Reply:
fixed